torch_xla/csrc/ops/native_dropout.cpp

```diff
@@ -0,0 +1,36 @@
+#include "torch_xla/csrc/ops/native_dropout.h"
+
+#include "torch_xla/csrc/helpers.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/xla_lower_util.h"
+
+namespace torch_xla {
+namespace {
+
+xla::Shape NodeOutputShape(const torch::lazy::Value& input) {
+  xla::Shape input_shape = GetXlaShape(input);
+  input_shape.set_element_type(xla::PrimitiveType::U8);
+  return xla::ShapeUtil::MakeTupleShape({input_shape, input_shape});
+}
+
+}  // namespace
+
+NativeDropout::NativeDropout(const torch::lazy::Value& input, float p,
+                             c10::optional<bool> train,
+                             const torch::lazy::Value& seed)
+    : XlaNode(torch::lazy::OpKind(at::aten::native_dropout), {input, seed},
+              [&]() { return NodeOutputShape(input); }, 2),
+      p_(p),
+      train_(train) {}
+
+torch::lazy::NodePtr NativeDropout::Clone(torch::lazy::OpList operands) const {
+  return torch::lazy::MakeNode<NativeDropout>(operands.at(0), p_, train_, operands.at(3));
+}
+
+XlaOpVector NativeDropout::Lower(LoweringContext* loctx) const {
+  xla::XlaOp input = loctx->GetOutputOp(operand(0));
+  xla::XlaOp seed = loctx->GetOutputOp(operand(3));
```

Comment:
same here, `operand(1)` for seed

Reply:
Thanks for pointing out! With this fix, the number of output mismatch issue got resolved.